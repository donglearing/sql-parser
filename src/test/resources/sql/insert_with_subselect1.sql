insert into T (a, b) select a, b from F;