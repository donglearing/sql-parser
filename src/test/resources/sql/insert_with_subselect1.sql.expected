INSERT INTO T (a, b) (SELECT a, b FROM F)